src/main/frontend/app/Show.js

```diff
@@ -1,35 +1,97 @@
-import React, {Component} from 'react';
+import React, {Component} from 'react'
+import ReviewContainer from './containers/ReviewContainer'
+import Review from './components/Review'
 
 
 class Show extends Component {
-    constructor(props){
-        super(props)
-        console.log(this.props.id)
-        this.state = {
-            fashionItem:{}
-        }
+  constructor(props){
+    super(props)
+    this.state = {
+      fashionItem:{},
+      reviews:[]
     }
+    this.addNewReview = this.addNewReview.bind(this)
+    this.getId = this.getId.bind(this)
+  }
+
+  componentDidMount(){
+    let id =this.getId()
+      fetch("/api/v1/fashion/" + id)
+      .then(resp => {
+          return resp.json()
+      })
+      .then(body => {
+          this.setState({fashionItem:body}) 
+      })
+
+      fetch("/api/v1/reviews/" + id)
+      .then(resp => {
+          return resp.json()
+      })
+      .then(body => {
+          this.setState({reviews:body.content}) 
+      })
+
+      fetch("api/v1/user")
+      .then(resp => {
+        return resp.json()
+      })
+      .then(body => {
+        this.setState({username:body})
+      })
+  }
```

Comment:
I think there is a better way to write this. I'm going to do some research and might ask you to refactor later. But I won't hold the merge up for it

Reply:
I agree. I was testing it and meant to delete that.

Reply:
I did some research (and Hieu helped actualize it) and it looks like what we want to do here is combine the JSON we're fetching on the backend to return an object which contains both the items and the reviews. 

I can follow up with you more in person.